hs628_tools/Visualization.py

```diff
@@ -0,0 +1,294 @@
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+import pandas as pd
+import sys 
+import plotly.plotly as py
```

Comment:
This script does not currently run for me, so I can't fully test this, but what I've read of plotly now seems to indicate that this code would only run within a live ipython notebook.  (There are ways to use plotly that are at least compatible with offline notebooks, but this module does not appear to use that structure.)  Have you guys tried running this from within a regular python script or checking it in an offline notebook?   A module that can only be used within an ipython notebook still has value, and wasn't prohibited by the assignment, but such constraints, and proper manner of use, need to be very clearly documented.